Packs/DuoAdminApi/ReleaseNotes/3_0_2.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### DUO Admin
+#  - Added command get_all_bypass_codes, which provides the ability to pull information from the bypass_code table.
+#     This identifies users in bypass mode, and allows for actions against these accounts
+#  - Added command get_all_admins, which provides the ability to get a list of the administrator accounts
+#  - Added command modify_admin_user, which allows the use of the update_admin command within the api
+#  - Added command modify_user, which allows the use of the update_user command within the api
+#  - Updated command get_all_users to include more of the fields available within the user table
```

Comment:
```suggestion
Added 3 commands:
    - ***get_all_bypass_codes***, which provides the ability to pull information from the bypass_code table, This identifies users in bypass mode, and allows for actions against these accounts.
    - ***get_all_admins***, which provides the ability to get a list of the administrator accounts.
    - ***modify_admin_user***, which allows the use of the update_admin command within the API.
    - ***modify_user***, which allows the use of the update_user command within the API.

- Updated command ***get_all_users*** to include more of the fields available within the user table.
```
Please [see](https://xsoar.pan.dev/docs/documentation/release-notes#examples) the correct format of the release notes